controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2285 @@
+    automated: yes
+    rules:
+      - var_selinux_state=enforcing
+      - selinux_state
+
+  - id: 1.7.1.5
+    title: Ensure no unconfined services exist (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - selinux_confinement_of_daemons
+
+  - id: 1.7.1.6
+    title: Ensure SETroubleshoot is not installed (Automated)
+    levels:
+      - l2_server
+    automated: yes
+    rules:
+      - package_setroubleshoot_removed
+
+  - id: 1.7.1.7
+    title: Ensure the MCS Translation Service (mcstrans) is not installed (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - package_mcstrans_removed
+
+  - id: 1.8.1.1
+    title: Ensure message of the day is configured properly (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - banner_etc_motd
+      - login_banner_text=usgcb_default
+
+  - id: 1.8.1.2
+    title: Ensure local login warning banner is configured properly (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - banner_etc_issue
+      - login_banner_text=usgcb_default
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5225
+  - id: 1.8.1.3
+    title: Ensure remote login warning banner is configured properly (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 1.8.1.4
+    title: Ensure permissions on /etc/motd are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_groupowner_etc_motd
+      - file_owner_etc_motd
+      - file_permissions_etc_motd
+
+  - id: 1.8.1.5
+    title: Ensure permissions on /etc/issue are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_groupowner_etc_issue
+      - file_owner_etc_issue
+      - file_permissions_etc_issue
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/7225
+  - id: 1.8.1.6
+    title: Ensure permissions on /etc/issue.net are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 1.8.2
+    title: Ensure GDM login banner is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - dconf_gnome_banner_enabled
+      - dconf_gnome_login_banner_text
+      - login_banner_text=usgcb_default
+
+  - id: 1.9
+    title: Ensure updates, patches, and additional security software are installed (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    related_rules:
+      - security_patches_up_to_date
+
+  - id: 1.10
+    title: Ensure system-wide crypto policy is not legacy (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - configure_crypto_policy
```

Comment:
After this PR gets merged, you can use the default policy for this rule and future policy for the next rule:
https://github.com/ComplianceAsCode/content/pull/7226

Reply:
Ah perfect! - I'll add this in now and we can just block merging of this PR until after #7226 gets merged.